app/controllers/votes_controller.rb

```diff
@@ -58,8 +63,13 @@ def destroy
     end
 
     if vote.destroy
+      post = Post.find(post.id)
```

Comment:
```suggestion
      post.reload
```

Reply:
@Taeir yeah, `.reload` is cleaner than `Post.find(post.id)`. I don't have a better solution for this either at this point other than recalculating the score manually for the affected post without actually triggering `recalc_score` - should save us a query - this could work, I think?

Reply:
Aah, I just looked at `recalc_score` and realized why Rails doesn't magically fix things for us. Its because we literally execute a raw SQL query there...

I guess that is done to let the database handle atomicity?